acmw-db-app/pages/forgotPassword.js

```diff
@@ -0,0 +1,89 @@
+import Head from 'next/head'
+import styles from '../styles/Reset.module.css'
+import React, { useState } from 'react';
+  
+class forgotPassword extends React.Component {
+    constructor() {
+    super();
+    this.state = {
+      input: {},
+      errors: {}
+    };
+     
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+     
+  handleChange(event) {
+    let input = this.state.input;
+    input[event.target.name] = event.target.value;
+  
+    this.setState({
+      input
+    });
+  }
+     
+  handleSubmit(event) {
+    event.preventDefault();
+  
+    if(this.validate()){
+        let input = {};
+        input["email"] = "";
+        this.setState({input:input});
+    }
+  }
+  
+  validate(){
+      let input = this.state.input;
+      let errors = {};
+      let isValid = true;
+
+      if (typeof input["email"] !== "undefined") {
```

Comment:
change this to `if(input["email"]) {` - that is equivalent to if not null and not undefined and not an empty string.

Reply:
if sticking with directly checking if the type is undefined, we should also directly check that the value is not null to be thorough